src/Frontend/Core/Engine/Block/Extra.php

```diff
@@ -163,23 +163,36 @@ public function execute()
     public function getAction()
     {
         // no action specified?
-        if ($this->action === null) {
+        if (null === $this->action) {
```

Comment:
I don't really like reversing the arguments in this conditional (called yoda conditions). It avoids assignments in your conditionals, but it has as tradeof that the readability of your code is less good. I think "$this->action is null" reads better then "null is $this->action" since you check the $this->action variable, not the null.